### PR TITLE
reduce font size in frontpage section and button

### PR DESCRIPTION
**Pull request by Gunkev**
In this PR I reduce the front page h1 font size and button and added transition on hover button

# Original Camel website

Screenshot_2020-03-21 Home - Apache Camel(3)

# Changes made

Screenshot_2020-03-21 Home - Apache Camel(2)

### CODE REVIEW

File: layouts/index.html [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {{ partial "header.html" . }}
 
 <header class="frontpage">
-  <h1>&#x1f680; Camel 3.1 is here! <wbr><a href="blog/Camel3-Whatsnew/">What's new?</a></h1>
+  <h1>&#x1f680; Camel 3.1 is here! <br><a href="blog/Camel3-Whatsnew/">What's new?</a></h1>
```
Comment (zregvart) [inline layouts/index.html]: This is intentional, please revert this change.

Comment (Gunkev) [reply layouts/index.html]: okay

**Comment by Gunkev** [comment]
> Any reason why the font needs to be changed? To me what we have looks good. This needs to be rebased to pick up the changes we made recently to the de

The font size seems to be tooo big on larger screen. Plsu transition was added to the button

**Comment by zregvart** [comment]
@Gunkev I understand that you think that the font size is too big. I think this is quite a common design pattern used today, have a look at some examples like: [Bootstrap](https://getbootstrap.com/) or [ReactJS](https://reactjs.org/) or [Netlify](https://www.netlify.com).

What I need from you is to argument why this needs to change, citing some design or usability reasons backed up with references, rather than to explain what changed.

**Comment by zregvart** [comment]
@Gunkev are you still working on this?

**Comment by Gunkev** [comment]
> @Gunkev are you still working on this?

**Comment by Gunkev** [comment]
@zregvart  no
